docs/kubernetes-cluster-management/label.md

```diff
@@ -0,0 +1,16 @@
+# Overview
+Labels are intended to be used to specify identifying attributes of objects that are meaningful and relevant to users, but do not directly imply semantics to the core system. Labels can be used to organise and to select subsets of objects. Labels can be attached to objects at creation time and subsequently added and modified at any time. Each object can have a set of key/value labels defined. Each Key must be unique for a given object.
+
+In order to take full advantage of using labels, they should be applied on every resource object.
+
+# Kubernetes Labels
+
+For each cluster we should use the below labels:
```

Comment:
The markup renders the labels all on one line, it could do with addtional MD formatting for readability.

Reply:
Fixed